sematic/ui/packages/common/src/pages/RunDetails/FunctionSection.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import Box from '@mui/material/Box';
+import { DateTimeLong, Duration } from 'src/component/DateTime';
+import Docstring from 'src/component/Docstring';
+import Headline from 'src/component/Headline';
+import ImportPath from 'src/component/ImportPath';
+import MoreVertButton from 'src/component/MoreVertButton';
+import PipelineTitle from 'src/component/PipelineTitle';
+import RunReferenceLink from "src/component/RunReferenceLink";
+import { SuccessStateChip } from 'src/component/RunStateChips';
+import Section from 'src/component/Section';
+import TagsList from 'src/component/TagsList';
+import theme from 'src/theme/new';
+
+const StyledSection = styled(Section)`
+    height: fit-content;
+    min-height: 200px;
+    position: relative;
+
+    &:after {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        height: 1px;
+        width: calc(100% + ${theme.spacing(10)});
+        margin-left: -${theme.spacing(5)};
+        background: ${theme.palette.p3border.main};
+    }
+    
+`;
+
+const StyledVertButton = styled(MoreVertButton)`
+    position: absolute;
+    right: 0;
+    top: 12px;
+    transform: translate(50%,0);
+`;
+
+const BoxContainer = styled(Box)`
+    display: flex;
+    flex-direction: row;
+    height: 35px;
+    align-items: center;
+    margin-bottom: ${theme.spacing(3)};
+
+    & .Info {
+        margin-left: ${theme.spacing(2)};
+
+        > div {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            column-gap: ${theme.spacing(2)};
+        }
+    }
+`;
+
+const RunStateContainer = styled(Box)`
+    flex-grow: 0;
+    width: 25px;
+`
+
+const ImportPathContainer = styled(Box)`
+    margin-bottom: ${theme.spacing(3)};
+`
+
+const StyledRunReferenceLink = styled(RunReferenceLink)`
+    font-size: ${theme.typography.fontSize}px ;
+`;
+
+const FunctionName = PipelineTitle
+
+const FunctionSection = () => {
+    return <StyledSection>
+        <Headline>Function Run</Headline>
```

Comment:
Currently the headline like a `Heading3`, and the docstring title looks like a `Title`:
![image](https://user-images.githubusercontent.com/1894533/233218294-dce646f3-7418-410d-9e9f-d8e52c4a6b5a.png)

The impression is that everything that comes before "Function header line" is an ignorable preamble. I actually thought it was a separate panel, and I was wondering why the Run section was so minimized that only part of its title was visible, but it seems that what I thought was the Run panel was the docstring.

The current way of rendering the docstring inside its own box is a lot clearer:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/1894533/233218567-29f725e9-285e-4560-bd10-9b69b056727e.png">

The different color contributes to this. Even if we want to reduce the number of employed colors, this looks like a case where it is required.

Please use a larger and non-greyed out font for the "FUNCTION RUN" panel title, and a box surrounding the docstring. Instead of cream, you can use a light grey.

Reply:
FUNCTION RUN should be in the same style as the widget headers in the left column.

What do people think of showing the docstring in a tab instead of below the run info? This would solve this problem as well as the fold-up/down problem. Also in a separate tab it would be easier to give it a special style like the cream background here.

Reply:
In the current implementation, the Run header which contains the docstring travels with you as you switch sub-tabs. The docstring is already included in raw format in the "Source" sub-tab. If we were to move it to a dedicated sub-tab, it would duplicate the information across 2 sub-tabs. I suspect this new dedicated sub-tab would not be used much by users.

To also include an answer to your comment [here](https://github.com/sematic-ai/sematic/pull/756#discussion_r1171973650), how about using a padded grey background (instead of cream) that would push the docstring contents downwards?

Reply:
+1 to `The docstring is already included in raw format in the "Source" sub-tab. If we were to move it to a dedicated sub-tab, it would duplicate the information across 2 sub-tabs. `

On:
> how about using a padded grey background (instead of cream) that would push the docstring contents downwards?

In the new design, the header section, including the docstring, is fixated at 200px tall (unless the docstring may be sometimes expanded). If we put too much content in the header section, like pushing the docstring contents downwards, we leave no room for the content below. 

I know the docstring part is jumping out, that is because it uses Markdown (e.g. `#1`) . [Here is an example](https://story.dev-usw2-sematic0.sematic.cloud/?path=/story/sematic-docstring--long) that doesn't use markdown, which doesn't look that popping out. So we can maybe interpret it as "If the docstring part sometimes pops up, it is because a user intentionally makes it Markdown, and should be fully responsible for its consequence."

Reply:
> If we put too much content in the header section, like pushing the docstring contents downwards, we leave no room for the content below.

That's my intention: for the user to see a docstring grey box starts (so if they want to see the contents, they can click on the expand button), but for the title in the docstring box to not overshadow the title of the Run panel itself.

> a user intentionally makes it Markdown, and should be fully responsible for its consequence.

In most cases, the user doesn't know what's going on.

Reply:
Per offline discussion, docstring does not provide much business value to users, hence removed.